src/common/utxobased/db/makeProcessor.ts

```diff
@@ -167,7 +167,8 @@ async function createOrOpen<T extends BaseType>(
     }
   } catch (err) {
     console.log('already exists', config)
-    if (!err.message.includes('already exists')) {
+    // eslint-disable-next-line no-extra-boolean-cast
+    if (!Boolean(err.message.includes('already exists'))) {
```

Comment:
No need to cast to Boolean but you might need to actually check that 'message' is in 'err'